examples/md-flexible/src/domainDecomposition/RegularGridDecomposition.cpp

```diff
@@ -192,10 +189,11 @@ void RegularGridDecomposition::exchangeHaloParticles(AutoPasType &autoPasContain
       }
     }
     // See documentation for _neighborDomainIndices to explain the indexing
-    int leftNeighbor = _neighborDomainIndices[(dimensionIndex * 2) % _neighborCount];
-    int rightNeighbor = _neighborDomainIndices[(dimensionIndex * 2 + 1) % _neighborCount];
-    sendAndReceiveParticlesLeftAndRight(particlesForLeftNeighbor, particlesForRightNeighbor, leftNeighbor,
-                                        rightNeighbor, haloParticles);
+    const int leftNeighbor = _neighborDomainIndices[(dimensionIndex * 2) % _neighborCount];
+    const int rightNeighbor = _neighborDomainIndices[(dimensionIndex * 2 + 1) % _neighborCount];
+    const auto haloParticlesThisDirection = sendAndReceiveParticlesLeftAndRight(
```

Comment:
It is not clear what this variable is from name.

Reply:
renamed it to `receivedHaloParticles`. Does this work better? 501f2024

Reply:
Yes this is good.